components/ide-metrics/pkg/errorreporter/error_reporter.go

```diff
@@ -45,6 +46,11 @@ type logErrorReporter struct {
 }
 
 func (r *logErrorReporter) Report(event ReportedErrorEvent) {
```

Comment:
@iQQBot Could you review please from IDE perspective? Do you see any other places which we scrub specially for logging or the rest is covered by common-go?